static/components/search.js

```diff
@@ -3,7 +3,7 @@ const search = instantsearch({
   apiKey: process.env.ALGOLIA_ADMIN_KEY,
   indexName: "apprenticeships",
   routing: true,
-  searchableAttributes: ["company", "description", "location"],
+  searchableAttributes: ["company", "description", "location", "image"],
```

Comment:
Should this be searchable (image)?

Reply:
How should I format this then?

Reply:
Sorry, I meant if the image was needed as a searchable attribute. Should the images be altering the search results?

Reply:
Oh nope - I can remove this 👍